drivers/ethernet/Kconfig.xlnx_gem

```diff
@@ -11,6 +11,7 @@ menuconfig ETH_XLNX_GEM
 	default y
 	depends on DT_HAS_XLNX_GEM_ENABLED
 	depends on !QEMU_TARGET || (QEMU_TARGET && NET_QEMU_ETHERNET)
+	select PINCTRL
```

Comment:
This driver does not use pinctrl either.

Reply:
Removed